scripts/index.js

```diff
@@ -335,14 +335,14 @@ Hooks.once('init', async function() {
     if(game.system.id === "pf2e"){
         Hooks.on("createItem", (document, options, userId) => {
             if(document.parent && (document.parent.token?.parent?.id || document.parent?.id)){
-                const data = pf2eItemToWorkflowData(document, "effect")
+                const data = pf2eItemToWorkflowData(document, "effect") || pf2eItemToWorkflowData(document, "condition")
```

Comment:
Here's adding conditions into the flow for pf2e.  Can remove this if you already have it somewhere else.